docs/reference/sem-command-line-tool.md

```diff
@@ -792,6 +804,22 @@ text editor.
 To learn more about the configuration options for projects, visit the
 [Projects YAML Reference](https://docs.semaphoreci.com/reference/projects-yaml-reference/) page.
 
+### sem apply 
+
+You can apply an existing YAML a project by using the `sem apply -f` command followed by the
+`name` of an project YAML file.
```

Comment:
Let's change this to:
> You can update an existing project by using the `sem apply -f` command followed by the valid `file path` to a project YAML file that contains the updates.


And then in the example below it can be:

> `sem apply -f [file_path]`